plugins/modules/controller_managed_device_group.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2024, Network to Code (@networktocode) <info@networktocode.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: controller_managed_device_group
+short_description: Create, update or delete managed device groups within Nautobot
+description:
+  - Creates, updates or removes managed device groups from Nautobot.
+notes:
+  - Tags should be defined as a YAML list
+  - This should be ran with connection C(local) and hosts C(localhost)
+author:
+  - Sven Winkelmann (@pugnacity)
+version_added: "5.4.0"
+extends_documentation_fragment:
+  - networktocode.nautobot.fragments.base
+  - networktocode.nautobot.fragments.tags
+  - networktocode.nautobot.fragments.custom_fields
+options:
+  name:
+    description:
+      - The name of the controller managed device groups
+    required: true
+    type: str
+  controller:
+    description:
+      - The name of the controller for this group
+    required: true
+    type: str
+  weight:
+    description:
+      - weight of the managed device group
+    required: false
+    type: int
+  parent:
+    description:
+      - parent group of the managed device group
+    required: false
+    type: str
+  devices:
+    description:
+      - devices which should be assigend to this group
+    type: list
+    elements: str
```

Comment:
I don't see `devices` as a field on the API schema:
<img width="367" alt="Screenshot 2024-11-20 at 4 14 02 PM" src="https://github.com/user-attachments/assets/9353a8d9-4cb0-411b-83aa-0c955edb50f4">

Can you actually assign devices directly to the managed device group in this fashion?